AdminCore.Services/EventService.cs

```diff
@@ -584,5 +583,16 @@ private static int AutoApproveEventsNotNeedingAdminApproval(EventTypes eventType
 
       return eventStatusId;
     }
+
+    private void CheckEventTypeAdminLevel(EventTypes eventTypes, int employeeId)
+    {
+      int eventTypeId = (int)eventTypes;
+      var levelRequired = DatabaseContext.EventTypeRepository.GetSingle(x => x.EventTypeId == eventTypeId).EmployeeRoleId;
+      var userLevel = DatabaseContext.EmployeeRepository.GetSingle(x => x.EmployeeId == employeeId).EmployeeRoleId;
+      if (levelRequired == (int)EmployeeRoles.SystemAdministrator && userLevel == (int)EmployeeRoles.User)
+      {
+        throw new Exception("User does not have the correct privileges to book this type of event.");
```

Comment:
I would tend to use more specific exception types, but I wouldn't block the PR because of it. If you're catching it later it can be troublesome.